### PR TITLE
NOJIRA | Added a new rule. | [Ray]

### PR DESCRIPTION
**Pull request by RaycatWhoDat**
Make a rule to yeet folks.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -96,11 +90,11 @@ client.on('ready', async () => {
...
     }
 
-    const markdownText = await getMarkdownText();
+    const markdownText = getMarkdownText();
     const segments = parseMarkdownToSegments(markdownText);
     const embedTexts = segmentsToEmbeds(segments);
-
-    await Promise.all((await channel.fetchMessages({ limit: 100 })).deleteAll());
+    
+    await Promise.all((await channel.fetchMessages({ limit: 20 })).deleteAll());
```
Comment (RaycatWhoDat) [inline main.js:97]: Is `20` too few?

File: main.js [hunk 2]
```diff
@@ -1,24 +1,18 @@
...
-const tokenFactory = () => process.env.DISCORD_TOKEN;
-const DISCORD_TOKEN = tokenFactory();
-
-const getMarkdownText = async () => (
-    new Promise((res, rej) => fs.readFile('README.md', 'utf8', (err, data) => {
-        if (err) return rej(err);
-        res(data);
-    }))
-);
+const getMarkdownText = () => fs.readFileSync('README.md', 'utf8');
```
Comment (RaycatWhoDat) [inline main.js:10]: We can't do anything without reading this so, why not make it synchronous?

File: main.js (hunk 1 above)
Comment (Samoxive) [reply main.js:97]: Current embed count is 9, I think 20 is fine.

File: main.js (hunk 2 above)
Comment (Samoxive) [reply main.js:10]: No particular reason, I was fascinated by async apis at the time I suppose.

**Comment by Samoxive** [comment]
LGTM, thanks for updating the script.

**Review by Samoxive — APPROVED** [review]

**Review by ecoerod — APPROVED** [review]

**Pull request merged**